src/store/load-data.js

```diff
@@ -1,10 +1,10 @@
 import { json } from 'd3-fetch';
-import config from '../../config';
-import getRandomPipeline from '../../utils/random-data';
-import formatData from '../../utils/format-data';
-import loremIpsum from '../../utils/data/lorem-ipsum.mock';
-import animals from '../../utils/data/animals.mock';
-import demo from '../../utils/data/demo.mock';
+import config from '../config';
+import getRandomPipeline from '../utils/random-data';
+import normalizeData from './normalize-data';
+import loremIpsum from '../utils/data/lorem-ipsum.mock';
+import animals from '../utils/data/animals.mock';
+import demo from '../utils/data/demo.mock';
```

Comment:
One thing i've noticed in this refactor PR is there is a lot of changing paths which can be hard to manage, as a recommendation I'd look to move to setting baseUrl in jsconfig.json (for CRA)

https://create-react-app.dev/docs/importing-a-component/#absolute-imports
https://code.visualstudio.com/docs/languages/jsconfig

Reply:
I'd be happy to revisit this at a later date if the project becomes more complex and deeply nested, but at this stage I'd prefer to avoid switching to absolute paths, in the interest of keeping the code environment simpler and more approachable for newbie devs - especially as it's open-source and we're inviting contributions.

Reply:
Why would it be less approachable for newbies? Introducing absolute imports doesn't break relative imports and is a learn once thing (no real overhead in complexity)

Reply:
When I first encountered it, I found it a little confusing, and spent some time trying to figure out where the path rename was being configured (as it's not evident at the location you first spot it). I did some googling for pros/cons and the learning curve was something listed as one of the cons.

Seeing as it would take >0 time to configure, it wouldn't solve the problem of changing paths in any of the changed paths in this PR, and this project doesn't have deeply nested paths, it doesn't seem necessary to me. I'm not saying it's a terrible idea, I just think the minuscule pros are outweighed by the minor cons.